File: app/src/main/java/me/devsaki/hentoid/workers/PurgeWorker.java
package me.devsaki.hentoid.workers;

import android.content.Context;

import androidx.annotation.NonNull;
import androidx.work.WorkerParameters;

import me.devsaki.hentoid.R;

public class PurgeWorker extends BaseDeleteWorker {
    public PurgeWorker(
            @NonNull Context context,
            @NonNull WorkerParameters parameters) {
        super(context, R.id.delete_service_purge, parameters);
    }
}

File: app/src/main/java/me/devsaki/hentoid/workers/PurgeWorker.kt
package me.devsaki.hentoid.workers

import android.content.Context
import androidx.work.WorkerParameters
import me.devsaki.hentoid.R

class PurgeWorker(context: Context, parameters: WorkerParameters) :
    BaseDeleteWorker(context, R.id.delete_service_purge, parameters)